bigquery_etl/glean_usage/templates/baseline_clients_first_seen_v1.query.sql

```diff
@@ -7,8 +7,10 @@ WITH
 _baseline AS (
   -- extract the client_id into the top level for the `USING` clause
   SELECT DISTINCT
-    sample_id,
-    client_info.client_id
+    client_info.client_id,
+    -- Some Glean data from 2019 contains incorrect sample_id, so we
+    -- recalculate here; see bug 1707640
+    udf.safe_sample_id(client_info.client_id) AS sample_id,
```

Comment:
Oh nevermind my main review comment, looks like it's being recalculated here.

Reply:
I think I slipped in the query.sql changes while you were reviewing, so this very well may not have been there when you first read through.